libs/application-generic/src/utils/process-node-attrs.ts

```diff
@@ -65,6 +85,10 @@ function processAttributes(
         attrs[attr] as string,
         attrs.fallback as string,
       );
+      if (!MAILY_FIRST_CITIZEN_VARIABLE_KEY.includes(flag)) {
```

Comment:
we do not want to update the `id` and `idShowKey`, because mainly hold the actual value in them and not the flag. as with other attributes. 
so for example `isUrlVariable=true/false` is a flag, while `id=payload.foo` is a value.